src/tpm2_wrap.c

```diff
@@ -3168,22 +3173,14 @@ int wolfTPM2_NVCreateAuth(WOLFTPM2_DEV* dev, WOLFTPM2_HANDLE* parent,
     #endif
         return rc;
     }
+    if (rc == TPM_RC_SUCCESS && alreadyExists)
```

Comment:
These if's without brackets always scare me a bit. Worry about someone adding a second line to the conditional and not adding brackets. Prefer to add brackets always, but up to you. Same comment applies elsewhere in this PR.